src/main/java/uk/gov/companieshouse/pscverificationapi/validator/UvidExistsValidator.java

```diff
@@ -3,21 +3,28 @@
 import static uk.gov.companieshouse.api.identityverification.model.UvidMatchResponse.AccuracyStatementEnum.FORENAMES_MISMATCH;
 import static uk.gov.companieshouse.api.identityverification.model.UvidMatchResponse.AccuracyStatementEnum.SURNAME_MISMATCH;
 
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
 import org.springframework.stereotype.Component;
 import org.springframework.validation.FieldError;
 import uk.gov.companieshouse.api.error.ApiErrorResponseException;
 import uk.gov.companieshouse.api.identityverification.model.UvidMatch;
 import uk.gov.companieshouse.api.identityverification.model.UvidMatchResponse;
+import uk.gov.companieshouse.api.model.psc.IndividualFullRecord;
 import uk.gov.companieshouse.api.model.pscverification.PscVerificationData;
+import uk.gov.companieshouse.api.model.transaction.Transaction;
+import uk.gov.companieshouse.api.psc.DateOfBirth;
+import uk.gov.companieshouse.api.psc.NameElements;
 import uk.gov.companieshouse.pscverificationapi.enumerations.PscType;
 import uk.gov.companieshouse.pscverificationapi.exception.IdvLookupServiceException;
+import uk.gov.companieshouse.pscverificationapi.exception.PscLookupServiceException;
+import uk.gov.companieshouse.pscverificationapi.helper.JsonHelper;
 import uk.gov.companieshouse.pscverificationapi.service.IdvLookupService;
 import uk.gov.companieshouse.pscverificationapi.service.PscLookupService;
 
 import java.text.MessageFormat;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.*;
```

Comment:
Wildcard import is not preferred. Recommend you tweak your IDE settings to use individual imports.